<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgebraicInputDialog</class>
 <widget class="QDialog" name="AlgebraicInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algebraic Curve Input</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_9"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_10"/>
   </item>
   <item row="12" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter in polynomials composed of terms in the format: ax^by^c, for integers a, b, c</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Examples: 1, x, 2x^3, -y^4, 2xy^2 - 3x^2y^5</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlgebraicInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlgebraicInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
